Source/relay/TourGuide/Items of the Month/2024/Everfull Dart Holster.ash

```diff
@@ -0,0 +1,53 @@
+//Everfull Dart Holster via TES
```

Comment:
There's some minor code formatting stuff I'd like to get fixed in here before we merge (extra brackets, mixed tabs/spaces, etc). Happy to push a commit myself to do it, lemme know if that's cool.

Reply:
totally cool, go ahead man